client/my-sites/site-settings/site-owner-transfer/confirmation-transfer.tsx

```diff
@@ -19,13 +24,20 @@ export function ConfirmationTransfer( {
 } ) {
 	const translate = useTranslate();
 	const progress = 0.3;
+	const [ isEmailSent, setIsEmailSent ] = useState< boolean >( false );
 	const [ error, setError ] = useState< { message?: string } | null >( null );
 	const { confirmTransfer } = useConfirmTransfer(
 		{ siteId },
 		{
-			onSuccess: () => {
-				recordTracksEvent( 'calypso_site_owner_transfer_confirm_success' );
-				page.redirect( `/sites?site-transfer-confirm=true` );
+			onSuccess: ( data ) => {
+				const { transfer, email_sent } = data as SiteTransferResponse;
+
+				if ( transfer ) {
+					recordTracksEvent( 'calypso_site_owner_transfer_confirm_success' );
+					page.redirect( `/sites?site-transfer-confirm=true` );
+				} else if ( email_sent ) {
+					setIsEmailSent( true );
```

Comment:
Can we also add `recordTracksEvent( 'calypso_site_owner_transfer_pending_invitation_sent' );`?

Reply:
Added!